app/controllers/albums.js

```diff
@@ -0,0 +1,11 @@
+const albumsService = require('../services/albumsService'),
+  logger = require('../logger');
+
+exports.list = (req, res, next) =>
+  albumsService
+    .list()
+    .then(albums => {
+      logger.info('Albums GET request succeeded');
+      return res.status(200).send({ albums: JSON.parse(albums) });
```

Comment:
I think JSON.parse is unnecessary

Reply:
I added it because the albums is coming in text form `"[{ id, album, ... }, ..., { ... }]`
so it was not being added to `res.body` but `res.text` instead.